spec/chewy/fields/base_spec.rb

```diff
@@ -402,33 +402,279 @@
 
     context 'objects and scopes', :orm do
       before do
+        stub_model(:location)
         stub_model(:city)
         stub_model(:country)
 
         City.belongs_to :country
+        Location.belongs_to :city
+        City.has_one :location, -> { order :id }
         Country.has_many :cities, -> { order :id }
+      end
 
-        stub_index(:countries) do
-          define_type Country do
-            field :id
-            field :cities do
+      context 'text fields type with and without ignore_blank flag' do
```

Comment:
Why this is inside the 'objects and scopes' context?

Reply:
I created a special section for ignore_blank option. Thank you